charts/infra-apps/examples/thanos.yaml

```diff
@@ -0,0 +1,12 @@
+thanos:
+  enabled: true
+  values:
+    existingObjstoreSecret: mysecret
+    storeGateway:
+      enabled: true
+    queryFrontend:
+      ingress:
+        enabled: true
+        hostname: thanos.example
```

Comment:
```
  ingress:
    ## Set to true to enable ingress record generation
    ##
    enabled: false

    ## Set this to true in order to add the corresponding annotations for cert-manager
    ##
    certManager: false

    ## When the ingress is enabled, a host pointing to this will be created
    ##
    hostname: thanos.local

    ## Ingress annotations done as key:value pairs
    ## For a full list of possible ingress annotations, please see
    ## ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md
    ##
    ## If certManager is set to true, annotation kubernetes.io/tls-acme: "true" will automatically be set
    ##
    annotations: {}
```

Reply:
From the default  values.